develop/blockchains/deployment/generate-chain-specs.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Generate Chain Specs and Wasm
```

Comment:
FYI, there are usually three different things when you deploy: 

- genesis head
- the wasm
- the chain spec (which is generated from the wasm)

```suggestion
title: Generate Genesis, Chain Specifications and Wasm
```